### PR TITLE
SAK-30505 Moved event fire to connections logic

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: profile2/impl/src/java/org/sakaiproject/profile2/logic/ProfileConnectionsLogicImpl.java [hunk 1]
```diff
@@ -277,6 +277,8 @@ public boolean confirmFriendRequest(final String fromUser, final String toUser)
 			log.info("User: " + fromUser + " confirmed friend request from: " + toUser); 
 			//send email notification
 			sendConnectionEmailNotification(fromUser, toUser, EmailType.EMAIL_NOTIFICATION_CONFIRM);
+            //post event
+            sakaiProxy.postEvent(ProfileConstants.EVENT_FRIEND_CONFIRM, "/profile/"+fromUser, true);
```
Comment (ottenhoff) [inline profile2/impl/src/java/org/sakaiproject/profile2/logic/ProfileConnectionsLogicImpl.java]: tabs/spaces

Comment (adrianfish) [inline profile2/impl/src/java/org/sakaiproject/profile2/logic/ProfileConnectionsLogicImpl.java]: Ye gods

Comment (adrianfish) [inline profile2/impl/src/java/org/sakaiproject/profile2/logic/ProfileConnectionsLogicImpl.java]: Sorted.

**Comment by ottenhoff** [comment]
LGTM

**Pull request merged**